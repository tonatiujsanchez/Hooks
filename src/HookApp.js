
const HookApp = () => {
  return (
    <>
      <h1>Hook App</h1>
    </>
  )
}

export default HookApp