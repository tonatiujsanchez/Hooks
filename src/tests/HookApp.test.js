import { shallow } from "enzyme"
import HookApp from "../HookApp"

describe('Pruebas en <HookApp />', () => { 

    test('HookApp debe mostrarse correctamente', () => { 
        const wrapper = shallow( <HookApp /> )
        
        expect(wrapper).toMatchSnapshot()
     })
 })
