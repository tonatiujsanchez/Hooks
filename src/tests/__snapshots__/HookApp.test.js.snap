// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <HookApp /> HookApp debe mostrarse correctamente 1`] = `
<Fragment>
  <h1>
    Hook App
  </h1>
</Fragment>
`;
